src/Services/Billing/Subscriptions.ts

```diff
@@ -0,0 +1,61 @@
+import Service from "../Service";
+import JsonResponse from "../../Interfaces/JsonResponse";
+import {Subscription} from "../../Models/Models";
+import {AxiosResponse} from "axios";
+
+class Subscriptions extends Service {
+
+    /**
+     * Check if current business is subscribed to entity
+     * TODO@CE: @CS what do you reckon we do for "boolean" responses
```

Comment:
You'll still have to wait for the round trip so I would possibly not care about the response and just go with the standard `then/catch` for pass or fail behaviour

Reply:
Yeah what i did actually, forgot to removed these :joy: